samtranslator/model/api/http_api_generator.py

```diff
@@ -489,14 +489,21 @@ def _set_default_authorizer(self, open_api_editor, authorizers, default_authoriz
                 path, default_authorizer, authorizers=authorizers, api_authorizers=api_authorizers
             )
 
-    def _get_authorizers(self, authorizers_config, default_authorizer=None):
+    def _get_authorizers(self, authorizers_config, enable_iam_authorizer=False):
         """
         Returns all authorizers for an API as an ApiGatewayV2Authorizer object
         :param authorizers_config: authorizer configuration from the API Auth section
-        :param default_authorizer: name of the default authorizer
+        :param enable_iam_authorizer: if true add an "AWS_IAM" authorizer
         """
         authorizers = {}
 
+        if enable_iam_authorizer:
+            authorizers["AWS_IAM"] = ApiGatewayV2Authorizer(is_aws_iam_authorizer=True)
+
+        # If all the customer wants to do is enable the IAM authorizer the authorizers_config will be None.
+        if authorizers_config == None:
```

Comment:
nit: Think it's better to use `is` instead of `==`

Reply:
ok, so `if authorizers_config is None` ? Why is `is` preferred over `==`?

Reply:
`==` is for equality, `is` is for identity.
It's a good practice when checking that something is not set, to use `is None`
You may actually want to use `if not authorizers_config:` here, as this will also check for values like empty list `[]` or empty object `{}` (which I think could be good checks here)
About is/==: https://stackoverflow.com/a/48120163

Reply:
Got it. Fairly easy change.